commons/passive/dreem/dreem_sleependpoints.avsc

```diff
@@ -0,0 +1,16 @@
+{
+  "type": "record",
+  "namespace": "org.radarcns.passive.dreem",
+  "name": "Dreem",
+  "doc": "AVRO Schema for DREEM sleep endpoints",
+  "fields": [
+    {"name": "studyId", "type":"string", "doc":"This is study number"},
+    {"name": "subjId", "type":"string", "doc":"Subject Identifier for the study. Format: xxx-yyy where xxx represents 3 digits for the site ID and yyy represents 3 digits for the participant ID"},
+    {"name": "recDateTime", "type":"string","logicalType":"timestamp-millis", "doc":"Date & time corresponding to the recording start. Format: (ISO8601) YYYY-MM-DD’T’HH:MM:SS[Z/+hhmm/-hhmm]"},
+    {"name": "offHead", "type":"int", "doc":"Percentage of the TRT during which the headband was on head"},
```

Comment:
What is TRT? Generally, we map percentages to a float with range 0-1.

Reply:
Is this value the same for a lot of consecutive records? Perhaps it would better fit to put offHead in a separate topic.